packages/widget-v2/src/utils/number.ts

```diff
@@ -34,7 +34,7 @@ export function limitDecimalsDisplayed(input: string | number) {
   return flooredAndLimitedDecimalPlacesNumber.toString();
 }
 
-export function calculatePercentageDifference(
+export function calculatePercentageChange(
```

Comment:
This function calculates the percentage change

$$
\frac{(V_2 - V_1)}{|V_1|} \times 100
$$

not the percentage difference

$$
\frac{|V_1 - V_2|}{\left(\frac{V_1 + V_2}{2}\right)} \times 100
$$

which is what caused the confusion/difference in calculation